### DISCUSSION TITLE
Hello 2 for 5 "Pouces Turing Smart Screen

### DISCUSSION THREAD

**Discussion by C2TouteBote** [mathoudebine/turing-smart-screen-python#459] [Themes]
[6 Themes.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/14072827/6.Themes.zip)
![Capture d'écran 2024-01-27 161829](https://github.com/mathoudebine/turing-smart-screen-python/assets/156389983/8fa5fb72-eaa6-491e-b661-5849e115832e)
![Capture d'écran 2024-01-27 161907](https://github.com/mathoudebine/turing-smart-screen-python/assets/156389983/a44afcc7-cada-415b-8afe-8550fc9241b6)

**Comment by C2TouteBote** [comment]
Multiple thèmes with a transparent thème or vidéos😊

**Comment by fatbikerider** [comment]
they are fantastic great work hows about making some vertical 5in

**Reply by C2TouteBote** [reply]
thank you for the vertical themes you must already have a project base then edit the images

**Comment by Suicidatroll** [comment]
Amigo, o programa da tela não reconhece esses temas, só reconhece .data, os seus são .theme
Como faço para converter?

**Reply by C2TouteBote** [reply]
Je ne sais pas du tout je suis désolé quelqu'un pourrait peut vous aider.
